app/src/main/kotlin/com/mapzen/erasermap/view/DynamicChildHeightRelativeLayout.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapzen.erasermap.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.RelativeLayout
+import java.util.ArrayList
+
+abstract class DynamicChildHeightRelativeLayout : RelativeLayout {
+
+  constructor(context: Context) : super(context) {}
```

Comment:
Interestingly enough it seems the curly braces `{}` can be safely deleted in Kotlin if the constructor body is empty and the app still compiles and runs fine ;)